templates/todo/api/csharp-cosmos-sql/TodoEndPointsExt.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.AspNetCore.Http.HttpResults;
+using Microsoft.AspNetCore.Mvc;
+using SimpleTodo.Api;
+
+namespace Todo.Api
+{
+    public static class TodoEndpointsExt
+    {
+        public static void MapTodoEndpoints(this WebApplication app)
+        {
+
+            app.MapGet("/lists", async (ListsRepository _repository, [FromQuery] int? skip = null, [FromQuery] int? batchSize = null) =>
+            {
+                return TypedResults.Ok(await _repository.GetListsAsync(skip, batchSize));
+            })
+                .Produces(200)
+                .WithOpenApi();
+
+
+            app.MapPost("/lists", async (ListsRepository _repository, [FromBody] CreateUpdateTodoList list) =>

```

Comment:
Similar to `[FromQuery]`, no need for `[FromBody]` here since it is implied for complex types that aren't registered in DI.